src/core/InstrumentFunctions.cpp

```diff
@@ -301,6 +301,7 @@ InstrumentFunctionArpeggio::InstrumentFunctionArpeggio( Model * _parent ) :
 	m_arpEnabledModel( false ),
 	m_arpModel( this, tr( "Arpeggio type" ) ),
 	m_arpRangeModel( 1.0f, 1.0f, 9.0f, 1.0f, this, tr( "Arpeggio range" ) ),
+	m_arpRepeatsModel( 1.0f, 1.0f, 8.0f, 1.0f, this, tr( "Note repeats" ) ),
```

Comment:
Right, the new coding style. I should have seen that one coming... :)
I'll fix these up in a new commit.

Reply:
Right. In the case above the added line adopts to the surrounding code. I think I'd rather keep it that way.

Reply:
I guess that stands true for all my suggestions :laughing: 
We need to have a clear rule whether the surrounding code matters when defining the code style to be used, because to be honest more than half the code base isn't compliant to our current guidelines atm. But I won't hold an approval on the PR because of that minor thing. Still trying to track that small issue mentioned earlier though, I'll let you know if I figure it out!